src/wovnio/wovnphp/API.php

```diff
@@ -3,6 +3,7 @@
 
   require_once DIRNAME(__FILE__) . '../../utils/request_handlers/RequestHandlerFactory.php';
 
+  use \Wovnio\Wovnphp\Logger;
   use Wovnio\Html\HtmlConverter;
```

Comment:
should the next 2 lines be `use \Wovnio...`?

Reply:
I think it is best to use `\` at begining of namespace here. I was afraid to change, but then I did it for an other file yesterday. I’ll change for more homogeneity :+1: